template-engine/chapter.md

```diff
@@ -1314,7 +1321,6 @@ how templates work, not part of the details of a particular template.  It feels
 cleaner to implement it like this than to have that code be part of the
 compiled template.
 
-
```

Comment:
I like having the major heading preceded by extra space to help them stand out as I move through the document.